community/import-tool/src/main/java/org/neo4j/tooling/ImportTool.java

```diff
@@ -210,7 +210,12 @@
                 "Page size in bytes, or e.g. 4M or 8k" ),
         LEGACY_STYLE_QUOTING( "legacy-style-quoting", Configuration.DEFAULT_LEGACY_STYLE_QUOTING,
                 "<true/false>",
-                "Whether or not backslash-escaped quote e.g. \\\" is interpreted as inner quote." );
+                "Whether or not backslash-escaped quote e.g. \\\" is interpreted as inner quote." ),
+        READ_BUFFER_SIZE( "read-buffer-size", org.neo4j.csv.reader.Configuration.DEFAULT.bufferSize(),
+                "<bytes, e.g. 10k, 4M>",
+                "Size of the buffer for reading input data. It has to be large enough to hold at least the " +
+                "biggest single value in the data. Actual memory used may be bigger than specified, " +
```

Comment:
Do you really think we should include `Actual memory used may be bigger than specified, given some amount of internal overhead` in option description here? isn't it the case for almost any kind of structures? Also it does not provide any hints or highlights how end user should react on that fact: should buffer be always 2k bigger or 5k bigger then expected bigger value?